hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -88,6 +89,21 @@ def test_single_job(self):
 
             assert self.read(output_file.name) == msg
 
+    @unittest.skipIf(not which("docker"), "docker command is missing")
```

Comment:
Let's not include this test for now. I know it leaves a testing hole (jobs with images in local mode), but that's a known testing hole. Safe handling of docker in docker will be enabled by short term goals we have around getting off of Docker. In the meantime, I'd rather not have unmaintained tests (I never run the tests locally).

Reply:
There is, apparently safe docker in docker now, without --priveleged. I haven't looked too far into it.

Reply:
Ok, we're going to have the exact same problem with regenie